actors/power/tests/power_actor_tests.rs

```diff
@@ -326,3 +326,113 @@ fn power_accounting_crossing_threshold() {
     h.expect_total_power_eager(&mut rt, &expected_total_below, &(&expected_total_below * 10));
     h.check_state();
 }
+
+#[test]
+fn all_of_one_miners_power_disappears_when_that_miner_dips_below_min_power_threshold() {
+    let small_power_unit = &StoragePower::from(1_000_000);
```

Comment:
nit: assert small_power_unit is less than power_unit so that this test will break in an obvious way if the minimum requirement is changed